include/blast/math/algorithm/arch/avx2/Tile.hpp

```diff
@@ -46,7 +47,7 @@ namespace blast :: detail
     BLAST_ALWAYS_INLINE void tile(xsimd::avx2 const& arch, StorageOrder traversal_order, std::size_t m, std::size_t n, FF&& f_full, FP&& f_partial)
     {
         size_t constexpr SS = SimdSize_v<ET>;
-        size_t constexpr TILE_STEP = 4;    // TODO: this is almost arbitrary and needs to be ppoperly determined
+        size_t constexpr TILE_STEP = 4;    // TODO: this is almost arbitrary and needs to be properly determined
```

Comment:
This is the step in 'columns' right? It depends on the architecture as well. E.g. the optimal tiles are different for AVX and AVX2, AVX512

Reply:
Yes, true. Also depends on data type (`float` or `double`). This is a reminder to do it in a more clever way. It would be also good to avoid duplicating the `tile()` function code between architectures.